csharp/test/Apache.Arrow.Tests/TypeTests.cs

```diff
@@ -96,6 +96,23 @@ public void TestStructGetFieldByName()
             Assert.True(struct_type.GetFieldByName("not_found") == null);
         }
 
+        [Fact]
+        public void TestListTypeConstructor()
+        {
+            var stringField = new Field.Builder().Name("item").DataType(StringType.Default).Build();
+            var stringType1 = new ListType(stringField);
+            var stringType2 = new ListType(StringType.Default);
+
+            var int64Field = new Field.Builder().Name("item").DataType(Int64Type.Default).Build();
+            var int64Type1 = new ListType(int64Field);
+            var int64Type2 = new ListType(Int64Type.Default);
```

Comment:
It seems that `int64` case is redundant.
Should we check both `string` and `int64` cases?

Reply:
Indeed, I remove it.